feature/google/src/test/java/BillingWrapperCommonTest.kt

```diff
@@ -168,77 +65,52 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun defersCallingSkuQueryUntilConnected() {
-        mockStandardSkuDetailsResponse()
+    fun defersCallUntilConnected() {
         every { mockClient.isReady } returns false
 
-        val productIDs = setOf("product_a")
-
-        wrapper.querySkuDetailsAsync(
-            ProductType.SUBS,
-            productIDs,
-            {
-                this@BillingWrapperTest.storeProducts = it
-            }, {
-                fail("shouldn't be an error")
-            })
+        val token = "token"
+        var consumePurchaseCompleted = false
+        wrapper.consumePurchase(token) { _, _ ->
+            consumePurchaseCompleted = true
+        }
 
-        assertThat(storeProducts).`as`("SKUDetailsList is null").isNull()
+        assertThat(consumePurchaseCompleted).isFalse
 
         every { mockClient.isReady } returns true
 
         billingClientStateListener!!.onBillingSetupFinished(billingClientOKResult)
 
-        assertThat(storeProducts).`as`("SKUDetailsList is not null").isNotNull
+        assertThat(consumePurchaseCompleted).isTrue
     }
 
     @Test
     fun canDeferMultipleCalls() {
-        mockStandardSkuDetailsResponse()
         every { mockClient.isReady } returns false
 
-        val productIDs = setOf("product_a")
+        val token = "token"
 
-        wrapper.querySkuDetailsAsync(
-            ProductType.SUBS,
-            productIDs,
-            {
-                this@BillingWrapperTest.skuDetailsResponseCalled += 1
-            },
-            {
-                fail("shouldn't be an error")
-            })
-        wrapper.querySkuDetailsAsync(
-            ProductType.SUBS,
-            productIDs,
-            {
-                this@BillingWrapperTest.skuDetailsResponseCalled += 1
-            },
-            {
-                fail("shouldn't be an error")
-            })
-        assertThat(skuDetailsResponseCalled).isZero()
+        var consumePurchaseResponseCalled = 0
+        wrapper.consumePurchase(token) { _, _ ->
+            consumePurchaseResponseCalled += 1
+        }
+        wrapper.consumePurchase(token) { _, _ ->
+            consumePurchaseResponseCalled += 1
```

Comment:
Since both of these are using the same variable, we don't know if both the completion blocks were called or only 1 of them 2 times. Maybe we can change to a different count for each or even a boolean?